### DISCUSSION TITLE
Group by feature

### DISCUSSION THREAD

**Discussion by binaryk** [binaryk/laravel-restify#395] [Ideas]
Adding a way to group by so the frontend could ask:

```
/api/restify/posts?group_by=status
```

**Comment by cristijora** [comment]
Things to consider if implemented
- Should be added to filter endpoints
- All group by fields should also be sortable
- When we group_by=status, it should also `sort=status` because we want to group all the the things that are matching the same statuses in this case.
- In most cases you'd want to group by a related entity e.g `group_by=creator.name` for example.
